harvester/sources/tests/extraction/test_publinova.py

```diff
@@ -94,3 +94,10 @@ def test_get_parties(self):
     def test_get_publisher_year(self):
         seeds = self.seeds
         self.assertEqual(seeds[0]["publisher_year"], 2023)
+
+    def test_get_doi(self):
```

Comment:
No tests with whitespaces?

Reply:
test 1 had whitespace in het midden van de doi, de code heeft het weggehaald. Moeten we whitespace in doi's houden?